### DISCUSSION TITLE
Why is there no `/api/v0/swarm/connect` command in the API?

### DISCUSSION THREAD

**Discussion by BoisterousCoder** [ferristseng/rust-ipfs-api#121] [Q&A] [answered]
I am new to IPFS and trying to understand some thing, so I might be understanding things wrong. However, it seems like using `/api/v0/swarm/connect` is a very basic feature for an IPFS app, yet I don't see an open issue for adding this command to the API. Is there some other way of achieving a similar goal with a different command, did I just miss the the method of running the command in the API, or is there some other reason an issue isn't already open that I am not thinking of. I can open an issue for if that's what is needed.

**Comment by ferristseng** [comment, marked as answer]
There's a TODO stub for this: https://github.com/ferristseng/rust-ipfs-api/blob/d2b88b2616c67c187efb98a8ac5b570dc14f6ca4/ipfs-api-prelude/src/api.rs#L2292.

The way the library is structured, this shouldn't be too difficult to add if it follows the general convention of the other API calls.

Feel free to open an issue or open a PR. Initially, this project was mostly driven by a personal project I had started, which is now defunct, so I am mostly relying on contributions to add new features. Thanks!

**Reply by iamjpotts** [reply]
What was the old project? Is it on github?

**Reply by ferristseng** [reply]
The old project was a web archiving project, but it isn't on github.
